herald/apps.py

```diff
@@ -13,4 +16,32 @@ class HeraldConfig(AppConfig):
     name = 'herald'
 
     def ready(self):
+        from .models import Notification
+        from herald import registry
+
         self.module.autodiscover()
+
+        try:
+            # add any new notifications to database.
+            for index, klass in enumerate(registry._registry):
+                if klass.verbose_name:
+                    verbose_name = klass.verbose_name
+                else:
+                    verbose_name = re.sub(
+                        r'((?<=[a-z])[A-Z]|(?<!\A)[A-Z](?=[a-z]))',
+                        r' \1',
+                        klass.__name__
+                    )
+
+                notification, created = Notification.objects.get_or_create(notification_class=klass.__name__)
```

Comment:
I think we should take the full python path to the notification class. What if you have notifications in 2 different apps with the same class name? Elsewhere in herald, we do `'{}.{}'.format(self.__class__.__module__, self.__class__.__name__)`. Should probably put that in a method on `BaseNotification` since we will be using it several places now.